src/richie/apps/demo/defaults.py

```diff
@@ -323,5 +323,6 @@
     "categories": {},
     "organizations": {"max_depth": 1},
     "persons": {"max_depth": 1},
+    # "programs": {"max_depth": 1},
```

Comment:
comment fail?

Reply:
Niet niet, i wrote it commented as a todo note, i reserved to include program in demo with the integration commit to come